pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -123,8 +123,15 @@ type Evaluator struct {
 	Interface
 }
 
+// Preempt returns a PostFilterResult carrying suggested nominatedNodeName, along with a Status.
+// The semantics of returned <PostFilterResult, Status> varies on different scenarios:
+// - <nil, Unschedulable/Error>. This denotes it's a transient/rare error that may be self-healed in future cycles.
```

Comment:
<nil, Unschedulable> also happens when there are already other pods being terminated by the preemptor. That's not an error.

Should the nominated node name be cleared in that case? If not, can you add an integration test to make sure it's not cleared?

Reply:
I think it's not cleared in this case, which should be correct.

Reply:
> Should the nominated node name be cleared in that case?

Nope, it shouldn't be cleared and that's the happy path.

> If not, can you add an integration test to make sure it's not cleared?

I think it's covered already. We have integration as well as e2e tests to cover the basics of preemption.

Reply:
<nil, Unschedulable> and <nil, Error> are the same - nil PostFilterResult (the first returned param) indicates we keep the nominatedNodeName unchanged. Any wordings you think is not clear?

Reply:
well, it's saying that "Unschedulable" is an error. Even if we don't clear the name in both scenarios, that comment is innacurate.

Reply:
I see the point. Reworded the comments.